python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -437,12 +470,18 @@ def current_size(self) -> int:
         return self.num_pending_actors() + self.num_running_actors()
 
     def num_running_actors(self) -> int:
-        return len(self._num_tasks_in_flight)
+        return len(self._running_actors)
+
+    def num_restarting_actors(self) -> int:
```

Comment:
We should also update `current_processor_usage/pending_processor_usage/progress_str` methods to reflect restarting actors. 
* current_processor_usage should include only alive actors.
* pending_processor_usage should include pending and restarting actors. 
* progress_str should display the numbers of pending and restarting actors.